docs/overview/learning-path-beginner.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Start Here! A guide for beginners making Office Add-ins
+description:  A recommended path for beginners through the learning resources for Office Add-ins.
+ms.date: 02/19/2020
+ms.topic: conceptual
+ms.custom: scenarios:getting-started
+localization_priority: Priority
+---
+
+# Start Here! A guide for beginners making Office Add-ins
+
+Thank your for your interest in Office Add-ins, the Office extensions that run cross-platform, including Office on the web, Windows, and Mac. Like many people who are exploring a new framework, your first question might be **"Where do I start?"**. So, we created this recommended sequence of resources.
```

Comment:
Most of this looks really good, I just have a lot of tone suggestions throughout. 
I think it's worth trying to get to the point faster, suggestion below: 
```suggestion
Want to get started building your own cross-platform Office extensions? The following steps show you what to read first, what tools to install, and recommended tutorials to complete.
```